assets/src/components/cd/clusters/runtime/columns.tsx

```diff
@@ -166,19 +166,23 @@ const colKubVersion = columnHelperRuntime.accessor((row) => row?.addonVersion, {
 })
 const colBlocking = columnHelperRuntime.accessor((row) => row?.addonVersion, {
   id: 'blocking',
-  header: 'Blocks k8s upgrade',
-  cell: ({ getValue }) => {
+  header: 'Blocks upgrade',
+  cell: function Cell({ getValue }) {
+    const theme = useTheme()
     const addonVersion = getValue()
 
     if (!addonVersion?.blocking) return null
 
     return (
-      <Tooltip label="Blocking">
+      <Chip severity="danger">
         <BlockedIcon
           color="icon-danger"
-          size={16}
+          size="12"
```

Comment:
are these size values not in the theme?  We should use the theme instead of magic numbers as much as possible

Reply:
Yeah I was debating that. The theme has values for spacing which could still be used but not size, felt weird semantically